app/furniture/marketplace/tag.rb

```diff
@@ -10,5 +10,10 @@ class Tag < Record
 
     attr_accessor :marketplace
     location(parent: :marketplace)
+
+    positioned
+
+    scope :without_group, -> { where(is_group: false) }
+    scope :ordered_tag_groups, -> { where(is_group: true).merge(order(position: :asc, updated_at: :desc)) }
```

Comment:
I don't think it is possible for two tags to have the same position, at least not within whichever we decide is the reasonable "scope". In which case it is not relevant to try to have a second ordering criterion.